Testscripts/Linux/SRIOV-VerifyVF-Connection.sh

```diff
@@ -25,6 +25,28 @@ fi
     exit 0
 }
 
+function version_lt() { test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" != "$1"; }
+
+MIN_VERSION_SUPPORTED="6.7"
+
+if version_lt $DISTRO_VERSION $MIN_VERSION_SUPPORTED ; then
+    LogErr "SRIOV is not supported for Distro's below $MIN_VERSION_SUPPORTED, Test skipped!"
+    SetTestStateSkipped
+    exit 0
+fi
+
+if [[ "$NIC_COUNT" -gt 1 ]];then
+    NIC_IPs=($(ssh root@"${VF_IP2}" "hostname -I | awk '{print $1}'"))
+    if [[ ${NIC_IPs[1]} == "" ]] || [[ ${NIC_IPs[2]} == "" ]];then
```

Comment:
check only two interfaces?

Reply:
changed the logic to verify whether all the Extra NICs are having IP